src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -1,117 +1,30 @@
-import Backbone from 'backbone';
 import Radio from 'backbone.radio';
 
 import App from 'js/base/app';
 
-import { SidebarView } from 'js/views/patients/patient/sidebar/sidebar_views';
+import { SidebarView, WidgetCollectionView } from 'js/views/patients/patient/sidebar/sidebar_views';
 
 export default App.extend({
   onBeforeStart({ patient }) {
     const currentOrg = Radio.request('bootstrap', 'currentOrg');
 
-    this.orgEngagementEnabled = currentOrg.getSetting('engagement');
+    this.widgetSettings = currentOrg.getSetting('widget_sidebar');
     this.patient = patient;
 
-    const widgets = this.getSidebarWidgets();
-
     this.showView(new SidebarView({
-      model: this.patient,
-      collection: widgets,
+      model: patient,
     }));
+    this.getRegion('widgets').startPreloader();
   },
   beforeStart({ patient }) {
```

Comment:
I don't we should preload or fetch here at all.  We should add the widget fetch to the bootstrap and he we'd just use  `Radio.request('entities', 'widget:collection', _.collectionOf(currentOrg.getSetting('widget_sidebar')));` and then `WidgetCollectionView` could go back to having a template